specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -3099,6 +3099,54 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/incidents/{incidentId}/entities": {
+      "post": {
```

Comment:
Why is it modeled as POST?

Reply:
From offline conversation “Entities are not a child resource of an incident (they can be independent resources, or nested under other resources as well). Therefore we couldn't create a GET endpoint nested under the incidents resource.”